### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by tnolden** [eclipse-edc/Connector#2248] [Q&A] [answered]
Hey everyone,
I'm trying to build the project on my windows pc. I always get this error:

` error: unknown tag: Spi
@Spi(value = "Auth services")
^
1 error
1 warning

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':spi:common:auth-spi:javadoc'.
> Javadoc generation failed. Generated Javadoc options file (useful for troubleshooting): 
DataSpaceConnector\spi\common\auth-spi\build\tmp\javadoc\javadoc.options'`

Has anyone an idea how i could fix this?

**Comment by wolf4ood** [comment, marked as answer]
Hi @tnolden 

which JDK version are you using?

probably you hit this regression 
https://bugs.openjdk.org/browse/JDK-8222091

meanwhile you can skip the javadoc phase by adding `-x javadoc` when building the project

**Reply by tnolden** [reply]
Thanks for your quick answer!

I use Java 17.0.5. 

With the skip the build was successfull, thanks!
